app/scss/main.scss

```diff
@@ -0,0 +1,11 @@
+@import "~font-awesome/scss/font-awesome.scss";
+
+$app-bg: #fff7a7;
+
+body {
+  background-color: $app-bg;
+}
+
+app-red * {
+  border: 2px solid red;
+}
```

Comment:
@loganlsr this is fine for now but could definitely use more styles in future iterations ;)